### PR TITLE
feat(pci-menu): PCI menu entries revamp

### PR DESCRIPTION
**Pull request by varun257**
### Description of the Change

add oui header tabs at parent
combine headers of billing and compute
add rename directive directly
remove children on sidebar menu

### Benefits

Helps rearranging pci menu in a better way

### Applicable Issues

resolves issue MBE-197

### CODE REVIEW

File: client/app/cloud/project/cloud-project.html [hunk 1]
```diff
@@ -4,15 +4,62 @@
...
     </oui-message>
 
+    <div class="cloud-project-compute-content" data-ng-if="!CloudProjectCtrl.loaders.project">
+        <header class="oui-page-header">
+            <div class="oui-page-header__container">
+                <div class="oui-page-header__content">
+                    <cloud-project-rename
+                        data-ng-if="CloudProjectCtrl.model.project.description"
+                        data-project-id="CloudProjectCtrl.model.project.project_id"></cloud-project-rename>
+                    <h2 class="oui-page-header__description" data-ng-bind="::CloudProjectCtrl.model.project.project_id"></h2>
```
[the remark this answers is not in the document]
Comment (varun257) [reply client/app/cloud/project/cloud-project.html]: `<cloud-project-rename>` creates `h1` tag for `title`

**Comment by FredericEspiau** [comment]
What out your commits, you could rebase to merge them all :)

**Comment by FredericEspiau** [comment]
I've asked to some people internally some explanation about the translations

**Comment by FredericEspiau** [comment]
Watch your commits, you should rebase on `develop`

File: client/app/onboarding/pci/pci.html [hunk 2]
```diff
@@ -0,0 +1,26 @@
+<div class="cui-page__content">
+    <oui-slideshow data-theme="pci">
+        <oui-slideshow-panel data-heading="{{ ::'pci_slideshow_heading1' | translate }}" data-picture="../../assets/images/onboarding/revamp-PCI.jpg">
+            <ul>
+                <li class="list-unstyled clearfix" data-translate="pci_slideshow_description_infra1"></li>
+                <li class="list-unstyled clearfix" data-translate="pci_slideshow_description_infra2"></li>
+            </ul>
```
Comment (marie-j) [inline client/app/onboarding/pci/pci.html]: https://ovh-ux.github.io/ovh-ui-kit-documentation/#!/ovh-ui-kit/list doesn't fit the needs ?

File: client/app/onboarding/pci/pci.html [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+            <ul>
+                <li class="list-unstyled clearfix" data-translate="pci_slideshow_description_management1"></li>
+                <li class="list-unstyled clearfix" data-translate="pci_slideshow_description_management2"></li>
+            </ul>
+        </oui-slideshow-panel>
+        <oui-slideshow-panel
+            data-heading="{{ ::'pci_slideshow_heading4' | translate }}"
+            data-picture="../../assets/images/onboarding/revamp-PCI.jpg"
+            data-ng-href="/"
+            data-label="{{ 'discover' | translate }}">
```
Comment (marie-j) [inline client/app/onboarding/pci/pci.html]:
```suggestion
             data-label="{{:: 'discover' | translate }}">
```

File: client/app/onboarding/pci/pci.html [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+        <oui-slideshow-panel data-heading="{{ ::'pci_slideshow_heading3' | translate }}" data-picture="../../assets/images/onboarding/revamp-PCI.jpg">
+            <ul>
+                <li class="list-unstyled clearfix" data-translate="pci_slideshow_description_management1"></li>
+                <li class="list-unstyled clearfix" data-translate="pci_slideshow_description_management2"></li>
+            </ul>
+        </oui-slideshow-panel>
+        <oui-slideshow-panel
+            data-heading="{{ ::'pci_slideshow_heading4' | translate }}"
+            data-picture="../../assets/images/onboarding/revamp-PCI.jpg"
+            data-ng-href="/"
```
Comment (marie-j) [inline client/app/onboarding/pci/pci.html]:
```suggestion
             data-href="/"
```
?

File: client/app/onboarding/pci/pci.less [hunk 5]
```diff
@@ -0,0 +1,121 @@
+.oui-slideshow {
+  width: rem-calc(570);
+
+  &_pci {
+    &:not(.is-illustrated) .oui-slideshow__header {
+      .oui-modal__close-button {
+        color: #fff;
```
Comment (FredericEspiau) [inline client/app/onboarding/pci/pci.less]: `oui-color-white`

File: client/app/onboarding/pci/pci.less [hunk 6]
```diff
@@ -0,0 +1,121 @@
...
+    }
+
+    .oui-slideshow__body,
+    .oui-slideshow__content,
+    .oui-slideshow__footer {
+      background-color: @oui-color-pacific;
+    }
+
+    .oui-slideshow-panel__picture-container {
+      background-color: #fff;
```
Comment (FredericEspiau) [inline client/app/onboarding/pci/pci.less]: `oui-color-white`

File: client/app/onboarding/pci/pci.less [hunk 7]
```diff
@@ -0,0 +1,121 @@
...
+    .oui-slideshow-panel.active:not(.left):not(.right) {
+      .oui-slideshow-panel__picture {
+        opacity: 1;
+        width: 100%;
+      }
+    }
+
+    .oui-slideshow-panel__container {
+      align-items: center;
+      color: #fff;
```
Comment (FredericEspiau) [inline client/app/onboarding/pci/pci.less]: `oui-color-white`

File: client/app/onboarding/pci/pci.js [hunk 8]
```diff
@@ -5,6 +5,8 @@ angular.module('managerApp').config(($stateProvider) => {
       views: {
         onboardingContent: {
           templateUrl: 'app/onboarding/pci/pci.html',
+          controller: 'pciSlideshowCtrl',
+          controllerAs: 'Ctrl',
```
Comment (marie-j) [inline client/app/onboarding/pci/pci.js]: `Ctrl` -> `$ctrl` ?

**Comment by jleveugle** [comment]
Hello @varun257,
Your pull request is now closed due to inactivity.
Feel free to re-open it if you have any update.
Anna Bot for OVH UX <3

**Comment by varun257** [comment]
Trying to rebase

**Pull request closed without merging**